src/sql/workbench/parts/queryHistory/browser/queryHistoryView.ts

```diff
@@ -71,12 +68,8 @@ export class QueryHistoryView extends Disposable {
 					e.queryInfo.selection[0].endLine,
 					e.queryInfo.selection[0].endColumn + 1));
 
-				// exapnd as required
+				// expand as required
 				let newNode = new QueryHistoryNode(text, this._connectionManagementService.getConnectionProfile(e.uri), new Date(), undefined, QueryStatus.Succeeded);
-				newNode.hasChildren = true;
-				if (text.length > 100) {
-					newNode.children = [new QueryHistoryNode(text, undefined, undefined, undefined, QueryStatus.Nothing)];
-				}
```

Comment:
Hrm, I don't know why it still thinks these changes need to be merged in. Anyways the removal of these lines here are the only changes I'm making (the rest is from the previous commit I just merged in)

Reply:
Ah - that's right since I did a squash merge into the feature branch it didn't know the other commits were already included. Merging in the latest changes from the upstream feature branch fixed it =)